rustls/src/error.rs

```diff
@@ -1,11 +1,40 @@
-use crate::msgs::enums::{AlertDescription, ContentType, HandshakeType};
+use crate::msgs::enums::{AlertDescription, ContentType, HandshakeType, KeyUpdateRequest};
 use crate::rand;
 
 use std::error::Error as StdError;
 use std::fmt;
 use std::sync::Arc;
 use std::time::SystemTimeError;
 
+/// A corrupt TLS message payload that resulted in an error.
```

Comment:
Style nit: probably better to move this below the `Error` definition, close to `PeerMisbehaved`.

Reply:
Good call, this has been moved down in the file.